packages/gallery/src/components/item/itemView.js

```diff
@@ -987,8 +987,9 @@ class ItemView extends React.Component {
       options.titlePlacement !== GALLERY_CONSTS.placements.SHOW_ON_HOVER &&
       !this.hasRequiredMediaUrl;
     const itemAriaRole = this.getItemAriaRole();
+    const Element = this.props.elementsOverride.item || 'div';
```

Comment:
should add '?' for this check. if elementOverride doesn't exist this line will access undefined.